### PR TITLE
Include getting GCP permissions in the state-viewer documentation.

### PR DESCRIPTION
**Pull request by nikurt**

### CODE REVIEW

File: test-utils/state-viewer/README.md [hunk 1]
```diff
@@ -37,14 +37,18 @@ obtaining this whole history is the following:
 
 * Patch https://github.com/near/near-ops/pull/591 to define your own GCP instance in project `rpc-prod`.
 * Make sure to change `machine-name` and `role` to something unique.
+* Make a Pull Request and ask Mario or Sandy for review.
```
Comment (chefsale) [inline test-utils/state-viewer/README.md:40]: Sandi* 😄

File: test-utils/state-viewer/README.md [hunk 2]
```diff
@@ -37,14 +37,18 @@ obtaining this whole history is the following:
...
+* Ask Mario or Sandy to grant you permissions to the GCP project `rpc-prod`.
```
Comment (chefsale) [inline test-utils/state-viewer/README.md:41]: Sandi* 😄

**Review by chefsale — APPROVED** [review]
LGTM, just fix my name please. Haha 👯

File: test-utils/state-viewer/README.md (hunk 1 above)
Comment (nikurt) [reply test-utils/state-viewer/README.md:40]: Sorry about that.

**Pull request merged**